libbeat/_meta/config.reference.yml

```diff
@@ -867,6 +867,10 @@ output.elasticsearch:
 #- name: field_name
 #  type: field_type
 
+# List of external fields.yml files to load.
+#setup.template.external_fields:
+#- /path/to/external/fields.yml
```

Comment:
To me the name kind of clashes with `setup.template.fields`. Like: which one has priority, does one replace the other, are the two files combined, why not have setup.template.fields accept a list of files.